rails/app/models/water_entry.rb

```diff
@@ -17,3 +17,9 @@ def self.units
     }
   end
 end
+
```

Comment:
Please try to keep the files tidy and not introduce irrelevant changes. You can set up your editor to automatically remove these extra new lines when you save a file.